### ISSUE TITLE
Error Progress Complete At 461 Files

### ISSUE THREAD

**Issue by reikarin** [ericyd/gdrive-copy#67]
### Are you requesting a feature or reporting a bug?
reporting a bug

### If reporting a bug, is there already an issue open for this same bug?
yes

### What is the current behavior?
copying folder error, progress completed when reach 461 files

### What is the expected behavior?
copying progress still running until quota limit reach

### Specify your

* operating system: Windows 10 Pro 65 Bit
* browser: Google Chrome 71.0.3578.98

**Comment by johnmcdowell** [comment]
Sure. Unfortunately, it might have made it worse. I just tried copying a folder with 1,062 google docs, and it copied exactly 100 files, marked the Copy Log "complete," and said "Total Files Copied" was 101 (instead of the actual 100 out of 1,062).

I went back and checked and the previous incomplete copies actually copied 460 (unless you also include the Copy Log, which I wouldn't have expected).

Hope that helps.
